README.md

```diff
@@ -97,6 +98,15 @@ define( 'WP_REDIS_HOST', 'master.ncit.ameaqx.use1.cache.amazonaws.com' );
 define( 'WP_REDIS_PORT', 6379 );
 ```
 
+Additional stream connection options for `phpredis` and `predis` connections can be defined using `WP_REDIS_TLS`
+
+```php
+define( 'WP_REDIS_TLS', [
+    'verify_host' => false,
+    'verify_host_name' => false
+]);
+```
+
```

Comment:
Now seeing the code, I think we need to change the name. What this constant really is are context options: https://www.php.net/manual/en/context.ssl.php

Let's call it `WP_REDIS_SSL_CONTEXT` to stick with the PHP naming.

Reply:
Changed